Example/Example/ContentView.swift

```diff
@@ -0,0 +1,105 @@
+import SwiftUI
+import GEOSwift
+
+struct ContentView: View {
+    @State private var modifiableGeometry = GeometryModification()
+    @State private var tempGeometry = try! Geometry(wkt: "POLYGON((35 10, 45 45.5, 15 40, 10 20, 35 10),(20 30, 35 35, 30 20, 20 30))")
```

Comment:
`tempGeometry` is always written to before it is read in each button action. That means it could just be a local constant in each of those methods rather than a property here.